src/main/java/de/tum/in/www1/artemis/domain/quiz/QuizQuestion.java

```diff
@@ -68,6 +68,13 @@ public abstract class QuizQuestion extends DomainObject {
     @JsonIgnore
     private QuizExercise exercise;
 
+    @ManyToOne
+    private QuizGroup quizGroup;
+
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
FetchType.LAZY does not work for ManyToOne. This will be loaded EAGER, which will lead to performance issues due to additional database queries, even if the value was never set. Can we avoid it?